zulipterminal/ui_tools/boxes.py

```diff
@@ -539,10 +539,15 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     topic = self.title_write_box.edit_text
 
                 if self.msg_edit_state is not None:
+                    if topic == self.msg_edit_state[1]:  # FIXME need strip?
```

Comment:
For a topic change from "Test" to "Test ", it says invalid prop mode. I think we need to strip here.